### DISCUSSION TITLE
You have weak internet connection or the server has problems. Try to refresh the page. Error code [404]

### DISCUSSION THREAD

**Discussion by ibuchan** [LiveHelperChat/livehelperchat#1900] [Q&A]
We have been using Live Helper Chat for some years and are upgrading to the latest version which we are looking to get running on PHP 8.1 with MySQL 5.7 on Windows.  We have tried a clean install of the latest version and that runs through with no issues but when operators log in they get message

You have weak internet connection or the server has problems. Try to refresh the page. Error code [404]

They can use the back end system fine but when users try to start a chat are always told nobody is online even when multiple operators are logged in.  

We have tried multiple clean installs and tried different servers but always the same thing.  Is there a specific setting in PHP or MySQL that is required for LHC to fix this?

**Comment by remdex** [comment]
Hi,

Well on my version I'm running 8.1php and some recent MySQL version also and no problems. I ques there is something with your env just.

I'm afraid only personal investigation could help you identify what you are missing...

**Comment by echodreamz** [comment]
Would like to add a "fix" for those experiencing this on IIS. By default, IIS limits to 32 URL segments, which causes the 404.20 error.

The fix is to navicate open the IIS manager and then the Configuration Editor

![image](https://github.com/LiveHelperChat/livehelperchat/assets/18668727/c17e9e45-2928-4d05-a4a4-771ca1c2d9b3)

Change the "maxUrlSegments" option to a higher number.
